04/app.js

```diff
@@ -1,5 +1,66 @@
-document.addEventListener('DOMContentLoaded', init);
+const apiUrl = `https://api.weatherbit.io/v2.0/current?key=2d15c1f36cfb4b6796484745262c485a&include=minutely&lang=pl&units=I`;
 
-function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+const form = document.querySelector(".form");
+
+// weather text elements
+const weatherLat = document.querySelector(".weather__lat");
+const weatherLng = document.querySelector(".weather__lng");
+const weatherSummary = document.querySelector(".weather__summary");
+const temperature = document.querySelector(".weather__temperature");
+
+form.addEventListener("submit", init);
+
+function init(e) {
+  e.preventDefault();
+
+  setLatLng();
+  getWeatherDesc();
+  getWeatherTemp();
```

Comment:
Czy nie dałoby się tego zrobić jednym zapytaniem? Teoretycznie to rozwiązanie trwa 2 razy dłużej ;)